test/prompt/invocation_layer/test_hugging_face.py

```diff
@@ -468,6 +468,49 @@ def test_stop_words_multiple_token(stop_words: List[str]):
     assert "health" not in result[0]
 
 
+@pytest.mark.unit
+def test_stop_words_criteria():
+    """
+    Test that StopWordsCriteria will check stop word tokens in a continuous and sequential order
+    """
+    stop_words = ["unambiguously"]
```

Comment:
This is unecessary too, since we're mocking the tokenizer it makes sense to mock this too. 
Passing it directly to `StopWordsCriteria`'s init would be fine.
Something like this would work for me.
```StopWordsCriteria(tokenizer=Mock(), stop_words=["mock data"])```